File: 58. Length of Last Word/solution.py
class Solution(object):
  def lengthOfLastWord(self, s):
    """
    :type s: str
    :rtype: int
    """
    count = 0
    index = len(s) - 1
    while index > -1:
      if s[index] == ' ' and count > 0:
        break
      if s[index] != ' ':
        count += 1
      index -= 1
    return count

File: 58. Length of Last Word/solution.cpp
class Solution {
public:
  int lengthOfLastWord(string s) {
    int count = 0;
    int index = s.size() - 1;
    while (index > -1){
      if (s[index] == ' ' && count > 0){
        break;
      }
      if (s[index] != ' '){
        count += 1;
      }
      index -= 1;
    }
    return count;
  }
};